src/main/java/brandkon/brand/BrandService.java

```diff
@@ -0,0 +1,42 @@
+package brandkon.brand;
+
+import brandkon.category.CategoryRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class BrandService {
+    BrandRepository repository;
+    CategoryRepository categoryRepository;
+
+    public BrandService(BrandRepository repository, CategoryRepository categoryRepository) {
+        this.repository = repository;
+        this.categoryRepository = categoryRepository;
+    }
+
+    List<BrandResponse> showAll(){
+       return repository.findAll()
+               .stream()
+               .map(brand -> new BrandResponse(
+                       brand.getId(),
+                       brand.getName(),
+                       brand.getImageUrl()
+               )).toList();
+    }
+    List<BrandResponse> showCategory(String name){
```

Comment:
```suggestion
    List<BrandResponse> findAllByCategory(String name){
```
함수명은 관습적으로 이런 식으로 지어요